packages/core/src/common/_mixins.scss

```diff
@@ -20,10 +20,10 @@ $pt-intent-text-colors: (
 ) !default;
```

Comment:
Nope, @pkwi and I looked into it. We actually only want to do this for the dark theme